scripts/ci_build.sh

```diff
@@ -37,10 +37,9 @@ echo "n_parallel=${n_parallel}"
 
 echo "ccache configuration"
 ccache --version
-ccache -M 5G
+#ccache -M 5G
```

Comment:
Why remove this?

Reply:
I commented it out as I was running the ci_build and it modified my local ccache storage size. It seemed to me that the ci_build script should not set it on travis, rather when you set up a shared ccache on travis (which appears to have been done), the size should be set at that point. In my mind, anyone could set this to whatever number they want, either low (causing unrelated PRs to become slower) or high (causing travis to have less disk space). Unless I misunderstand something about the travis build config, that is.

Reply:
I don't mind leaving it in if there is some objection

Reply:
@valeryz @KaustubhShamshery

Reply:
let's revert this @n-hutton and handle in another PR, really has nothing to do with the purpose of this PR.